test/e2e/dlb/dlb.go

```diff
@@ -45,9 +46,9 @@ func describe() {
 	f := framework.NewDefaultFramework("dlbplugin")
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
-	kustomizationPath, err := utils.LocateRepoFile(kustomizationYaml)
-	if err != nil {
-		framework.Failf("unable to locate %q: %v", kustomizationYaml, err)
+	kustomizationPath, errFailedToLocateRepoFile := utils.LocateRepoFile(kustomizationYaml)
+	if errFailedToLocateRepoFile != nil {
+		framework.Failf("unable to locate %q: %v", kustomizationYaml, errFailedToLocateRepoFile)
```

Comment:
So there's no logical change, just the names of the variables change?
If that's the case, I think we should revert back to the original form. I feel the explicit error name is not needed. It's clear from the context.

Reply:
Mikko also asked about this problem.

If you see my commit message, the reason why i changed it to `errFailedToLocateRepoFile` is because of the parts that come after this declaration and still do declaring in the form `X, err := Y`.
This makes linter complains saying it shadows the declaration in the place where I made changes in the commit.

So, we need to choose
1. name the first declaration of 'err' to have another name (that's what I did in this commit)
2. name all the other 'err' to another name.
3. make the linter not complain about this.!

Reply:
But.... idk why, for some reason... it does not reproduce any more..!
Let me push again without this commit, and see how the ci/cd linter reacts.!

Reply:
Ok, I see. Then I'm fine with this version.

Reply:
Ah, yeah still it happens.
> test/e2e/sgx/sgx.go:68:12: shadow: declaration of "err" shadows declaration at line 51 (govet)
                podList, err := e2epod.WaitForPodsWithLabelRunningReady(ctx, f.ClientSet, f.Namespace.Name,
                         ^
test/e2e/sgx/sgx.go:85:7: shadow: declaration of "err" shadows declaration at line 51 (govet)
                        if err := utils.WaitForNodesWithResource(ctx, f.ClientSet, "sgx.intel.com/epc", 150*time.Second); err != nil {
                           ^
make: *** [Makefile:87: lint] Error 1

Is it ok if I put the `errFailedToLocateRepoFile` for the first declarations?

Reply:
It fails for you locally? It seems to pass on GH's link.

Reply:
Actually the succeeded test contains
`errPodNotRunning` for sgx test.

One other reason why we need to have just another declaration is because of the matter of 'multiple processes' that we have to consider (and still need fixes in many other places).

If we still have 'err' and do not declare there again, it uses the firstly declared one, which means, it may cause some race condition about using err in different processes.

https://github.com/intel/intel-device-plugins-for-kubernetes/pull/1470/files#diff-89c583b2a40dff988d0d71e9e6cd0c4b192c287a11e63afebb7ffc48bfc234b6R85-R93

Reply:
@tkatila Sorry Tuomas, I didn't understand what you meant clearly.
Do we keep it as `err` or is it ok to change them to `errPodNotRunning`?

Reply:
Sorry for the trouble. I didn't understand the context. Please bring the renaming commit back and I'll approve.